scripts/get-gee-build-environment.sh

```diff
@@ -0,0 +1,430 @@
+            libc6 libc6-dev libcap-dev libfreetype6 libfreetype6-dev \
+            libgdbm-dev libgeos-dev libgeos++-dev libgif-dev libgtest-dev \
+            libjpeg-dev libjpeg8-dev libmng-dev libogdi3.2-dev \
+            libperl4-corelibs-perl libpng12-0 libpng12-dev libpq-dev \
+            libproj-dev libstdc++6 libtool libx11-dev libxcursor-dev \
+            libxerces-c-dev libxft-dev libxinerama-dev libxml2-dev \
+            libxml2-utils libxrandr-dev openssl \
+            python-dev python2.7 python2.7-dev python-imaging \
+            python-setuptools rsync scons software-properties-common swig \
+            wget xorg-dev zlib1g-dev || return 1
+         apt-get $ASSUME_YES_PACKAGE_MANAGER_PARAMETER install python-pexpect \
+            || return 1
+
+        # Install Git:
+        # add-apt-repository ppa:git-core/ppa || return 1
+        # apt-get $ASSUME_YES_PACKAGE_MANAGER_PARAMETER update || return 1
+         apt-get $ASSUME_YES_PACKAGE_MANAGER_PARAMETER install git || return 1
+
+        install_git_lfs
+    }
+fi
+
+print_command_line_help()
+{
+    cat <<MSG
+
+${SELF_NAME} [-r] [-b <branch_name>] [-p <dir>] [-d <dir>] [-c] [-l] [-y] [-h]
+
+    -p|--package-build-dir <dir>
+        Directory under which to build packages (.rpm, .deb) that are not
+        provided by the distribution we're running on.
+    
+    -d|--package-download-dir <dir>
+        Directory under which to download packages (.rpm, .deb) which are not
+        in package repositories.
+    
+    -c|--clean-package-build-dir
+        Specify '--clean-package-build-dir=no' to not delete the package build
+        directory before exiting.
+    
+    -l|--clean-package-download-dir
+        Specify '--clean-package-download-dir=no' to not delete the package
+        download directory before exiting.
+    
+    -y|--yes|--assume-yes
+        Specify '--assume-yes' to answer yes to all prompts. (Used when
+        installing packages.)  Use this option to run without user
+        interaction.
+    
+    -h|--help
+        Show this help message and exit.
+
+MSG
+}
+
+# Parse command-line arguments:
+while [[ "$#" -gt 0 ]]; do
+    case "$1" in
+        -p|--package-build-dir)
+            PACKAGE_BUILD_DIR="$2"
+            shift
+            ;;
+        -p=*|--package-build-dir=*)
+            PACKAGE_BUILD_DIR="${1#*=}"
+            ;;
+        -d|--package-download-dir)
+            PACKAGE_DOWNLOAD_DIR="$2"
+            shift
+            ;;
+        -d=*|--package-download-dir=*)
+            PACKAGE_DOWNLOAD_DIR="${1#*=}"
+            ;;
+        -c|--clean-package-build-dir)
+            CLEAN_PACKAGE_BUILD_DIR="yes"
+            ;;
+        -c=*|--clean_package_build_dir=*)
+            if is_string_value_false "${1#*=}"; then
+                CLEAN_PACKAGE_BUILD_DIR=""
+            else
+                CLEAN_PACKAGE_BUILD_DIR="yes"
+            fi
+            ;;
+        -l|--clean-package-download-dir)
+
```

Comment:
Minor: Looks like we could do without the empty line.

Reply:
Removed. Had those commented out but didn't get around to removing. Took out empty line.